css/index.css

```diff
@@ -172,4 +172,258 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
-}
\ No newline at end of file
+}
+
+@media(max-width: 768px){
+
+	.container {
+	    width: 100%;
+			padding:0 20px;
+	}
+
+	header {
+		flex-flow: row wrap;
+		justify-content:center;
+		width:100%;
+	}
+
+
+  header nav {
+		display:flex;
+		justify-content: around;
+		margin-top:20px;
+		width:760px;
+  }
+
+  header nav a {
+    padding: 20px;
+  }
+
+	header .logo {
+			/* outline: 2px red solid; */
+			margin:30px 0 0 0;
+			align-self: center;
+			order:-1;
+	}
+
+	.cta img{
+		display:none;
+	}
+
+	.cta {
+	    display: flex;
+	    margin-top: 50px;
+	    justify-content: center;
+			padding-right:20px;
+	}
+
+	.cta .cta-text {
+	    font-size: 72px;
+	    text-align: center;
+	    letter-spacing: 5px;
+	    display: flex;
+	    justify-content: center;
+	    flex-flow: column;
+	    align-items: center;
+	    margin: 0 0 0 0;
+			/* padding: 20px 20px 0 0; */
+	}
+
+	.cta .cta-text button {
+	    border: 1px double black;
+	    font-size: 16px;
+	    background: white;
+	    width: 170px;
+	    height: 32px;
+	    margin-top: 10px;
+	    cursor: pointer;
+	}
+
+	.cta .cta-text button:hover {
+	    background: black;
+	    color: white;
+	}
+
+	/* Main Content Styles */
+
+	.main-content {
+	   border-top: 2px solid black;
+	   border-bottom: 2px solid black;
+	   margin: 75px 0 35px 0;
+	   padding: 35px 0;
+	   font-size: 16px;
+	   line-height: 18px;
+	}
+
+	.main-content .top-content {
+	    display: flex;
+	    justify-content: space-around;
+	}
+
+	.main-content .middle-img {
+		    display: flex;
+				width:100%;
+		    margin: 30px auto;
+		}
+
+	.main-content .bottom-content {
+	    display: flex;
+	    justify-content: space-around;
+	}
+
+	.main-content .bottom-content p {
+	    padding-right: 20px;
+	}
+
+	/* Contact Styles */
+
+	.contact p {
+	    margin: 15px 0;
+	}
+
+	/* Footer Styles */
+
+	footer {
+	    text-align: center;
+	    margin: 40px 0 20px;
+	}
+
+}
+
+@media(max-width: 400px){
+	.container {
+	    width: 100%;
+			padding: 0 0 0 0;
+	}
+
+	header {
+		flex-flow: column;
+		justify-content:space-between;
+		width:100%;
+	}
+
+
+  header nav {
```

Comment:
I see whats happening, I was following the fact that the buttons do not center themselves in the smallest view.
Not of any importance though you fulfilled the media breakpoint very well.